pkg/okteto/kubetoken/cache.go

```diff
@@ -0,0 +1,122 @@
+package kubetoken
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/okteto/okteto/pkg/log"
+	authenticationv1 "k8s.io/api/authentication/v1"
+)
+
+const cacheIsCorrupted = "cache is corrupted"
```

Comment:
If we want to have it as a comparable error to use errors.Is on this we should define it as `errors.New("cache is corrupted")` and var. After that you can compare it on your tests and in other aprts of the code